[Power Toys > PowerToys Run plugins] Time and Date plugin: Add docs for custom formats (#5334)
MicrosoftDocs/windows-uwp: hub/powertoys/run.md

- `) calendar week::3/27/2022` to show the calendar week for a date value.
- `) unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.

#### Custom formats

> The plugin has a setting were you can define custom formats. The setting is a multiline text box were you have ta add one format per line.

Reviewer: ```suggestion
The plugin includes a setting where custom formats can be defined. Custom formats are entered in a multiline text box which accepts one format per line.
```

> The syntax is `<format name>=<format pattern>` for using the local time and `<format name>=UTC:<format pattern>` for using the Universal Time Convention (UTC).

Reviewer: Are these blocks intended to be separate paragraphs? If so, you should add an additional line break after each. Otherwise, they'll be combined into a single large paragraph.

Author: I will add a line break.

> Supported are all [standard patterns](https://learn.microsoft.com/dotnet/standard/base-types/custom-date-and-time-format-strings) and the following ones:

Reviewer: ```suggestion
The supported formats include all [standard patterns](https://learn.microsoft.com/dotnet/standard/base-types/custom-date-and-time-format-strings) and the following:

```

> - `DOW`: Number of the day in the week.

Reviewer: One more thought... Longer bulleted lists sometimes work better as tables. Let me know what you think.

Author: This is a bullet list currently?!

Reviewer: Yes... and I'm suggesting that it might be better as a table.

Author: From the layout in this case I like to keep the list.

Reviewer: OK, the [style guide](https://learn.microsoft.com/style-guide/scannable-content/lists) recommends "A list should have at least two items but no more than seven items.", but this isn't much longer than that.

- `) calendar week::3/27/2022` to show the calendar week for a date value.
- `) unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.

#### Custom formats
The plugin includes a setting where custom formats can be defined. Custom formats are entered in a multiline text box which accepts one format per line.

**Please use the following syntax:**

- `<Format name>=<Format pattern>` for using the local time.
- `<Format name>=UTC:<Format pattern>` for using the Universal Time Convention (UTC).

> [!NOTE]
> - `Format name`: Every charter except the equal sign is supported.
> - `Format pattern`: You can escape the pattern and the backslash itself as text by using a backslash as prefix.

**Examples:**

- `MyFormat=dd-MMMM-yyyy`
- `MySecondFormat=dddd (Da\y nu\mber: DOW)`
- `MyUtcFormat=UTC:hh:mm:ss`

**Supported format pattern:**

| Format pattern | Description |
| :--- | :--- |

> | Standard pattern like `hh:mm:ss`. |  Please see this [page](https://learn.microsoft.com/dotnet/standard/base-types/custom-date-and-time-format-strings) for more information.

Reviewer: This will fix the build suggestion.

```suggestion
| Standard pattern like `hh:mm:ss`. |  Please see this [page](/dotnet/standard/base-types/custom-date-and-time-format-strings) for more information.
```
